README.md

```diff
@@ -248,6 +248,7 @@ Validators support default `errorText` messages in these languages:
 - Turkish (tr)
 - Ukrainian (uk)
 - Vietnamese (vi)
+- Latvian (lv)
```

Comment:
Can move to follow the alfabethical order? Thanks!

Reply:
Done